src/Plugin.php

```diff
@@ -133,12 +135,10 @@ public function process(Event $event): void
 		file_put_contents($generatedConfigFilePath, $generatedConfigFileContents);
 		$io->write('<info>phpstan/extension-installer:</info> Extensions installed');
 
-		if ($oldGeneratedConfigFileHash === md5($generatedConfigFileContents)) {
-			return;
-		}
-
-		foreach (array_keys($installedPackages) as $name) {
-			$io->write(sprintf('> <info>%s:</info> installed', $name));
+		if ($io->isVerbose()) {
```

Comment:
previously, the logging was not repeatable (based on if file hash has changed or not), now it is repeatable and installed packages are dumped only if verbose logging is enabled